docs/code-examples/text_log_integration.cpp

```diff
@@ -0,0 +1,54 @@
+/// Shows integration of Rerun's `TextLog` with the C++ Loguru logging library
+/// (https://github.com/emilk/loguru).
+
+#include <loguru.hpp>
+#include <rerun.hpp>
+
+void loguru_to_rerun(void* user_data, const loguru::Message& message) {
+    // NOTE: `rerun::RecordingStream` is thread-safe.
+    const rerun::RecordingStream* rec = reinterpret_cast<const rerun::RecordingStream*>(user_data);
+
+    rerun::TextLogLevel level;
+    if (message.verbosity == loguru::Verbosity_FATAL) {
```

Comment:
Can't do switch case because it's a string?

Reply:
I had it as a switch initially (loguru verbosity is an integer), but it became so verbose, since every case becomes at least found lines:

```
case loguru::Verbosity_FATAL: {
    level = …
} break;

```

i.e. - the `if-else` chain reads better in the end